### PR TITLE
Issue/remove subspec

### PR DESCRIPTION
**Pull request by SergioEstevao**
Removes the use of subspecs and makes sure that calls that are blocked in extension are not used, or used correctly.

### CODE REVIEW

File: WordPressCom-Stats-iOS/Private/ExtensionUtils.m [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#import "ExtensionUtils.h"
+
+@implementation ExtensionUtils
+
```
Comment (nheagy) [inline WordPressCom-Stats-iOS/Private/ExtensionUtils.m]: This looks good, in that I think it'll work and putting it in an extension is great. Only thing I'd like to see is a comment to explain why we've done this, in the hopes that at some future date we can remove it.

File: WordPressCom-Stats-iOS/UI/InsightsTableViewController.m [hunk 2]
```diff
@@ -378,7 +379,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                         WPStatsViewController *statsViewController = (WPStatsViewController *)self.navigationController;
                         [self.statsDelegate statsViewController:statsViewController openURL:action.url];
                     } else {
-                        [[UIApplication sharedApplication] openURL:action.url];
+                        [ExtensionUtils openURL:action.url fromController:self];
```
Comment (nheagy) [inline WordPressCom-Stats-iOS/UI/InsightsTableViewController.m]: The nice thing is that `ExtensionUtils` kind of explains what this is for (ie compatibility with app extensions). 100% up to you, but if it were even more on-the-nose with its name maybe my comment… comment above would be irrelevant. Like "AppExtensionUtils"?

Just a thought. Don't let this comment stop you from merging if you hate the idea.

File: WordPressCom-Stats-iOS.podspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "WordPressCom-Stats-iOS"
-  s.version      = "0.7.8"
+  s.version      = "0.8.0"
```
Comment (astralbodies) [inline WordPressCom-Stats-iOS.podspec]: Minor version bump 👍

File: WordPressCom-Stats-iOS.podspec [hunk 4]
```diff
@@ -16,25 +16,14 @@ Pod::Spec.new do |s|
...
   s.source       = { :git => "https://github.com/wordpress-mobile/WordPressCom-Stats-iOS.git", :tag => s.version.to_s }
-  # s.source_files  = "WordPressCom-Stats-iOS", "WordPressCom-Stats-iOS/**/*.{h,m,swift}"
-  # s.private_header_files = "WordPressCom-Stats-iOS/Private/*.h"
+  s.source_files  = "WordPressCom-Stats-iOS", "WordPressCom-Stats-iOS/**/*.{h,m,swift}"
+  s.private_header_files = "WordPressCom-Stats-iOS/Private/*.h"
+  s.resource_bundle = { 'WordPressCom-Stats-iOS' => ['WordPressCom-Stats-iOS/UI/*.storyboard', 'WordPressCom-Stats-iOS/UI/*.xib', 'WordPressCom-Stats-iOS/Resources/*.otf', 'WordPressCom-Stats-iOS/Resources/*.png'] }
   s.exclude_files = "WordPressCom-Stats-iOS/Exclude"
   s.prefix_header_file = "WordPressCom-Stats-iOS/WordPressCom-Stats-iOS-Prefix.pch"
 
   s.requires_arc = true
```
Comment (astralbodies) [inline WordPressCom-Stats-iOS.podspec]: I think we may also wish to specify Swift 2.3 in this podspec for now to be totally correct. I think this works: 

```ruby
s.pod_target_xcconfig = { 'SWIFT_VERSION' => '2.3' }
```

**Comment by SergioEstevao** [comment]
@astralbodies ready for another look.

**Pull request merged**